src/app/blog/_components/post-body.tsx

```
@@ -0,0 +1,16 @@
+import markdownStyles from "./markdown-styles.module.css";
+
+type Props = {
+  content: string;
+};
+
+export function PostBody({ content }: Props) {
+  return (
+    <div className="mx-auto max-w-3xl text-lg tracking-tight sm:tracking-normal md:max-w-4xl md:text-xl md:tracking-wide lg:text-2xl lg:tracking-wider xl:tracking-widest">
+      <div
+        className={markdownStyles["markdown"]}
+        dangerouslySetInnerHTML={{ __html: content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)